Cake.Recipe/Content/packages.cake

```diff
@@ -167,6 +167,8 @@ BuildParameters.Tasks.PublishReleasePackagesTask = Task("Publish-Release-Package
     PushChocolateyPackages(Context, true, chocolateySources);
 
     PushNuGetPackages(Context, true, nugetSources);
+
+    BuildParameters.PublishReleasePackagesWasSuccessful = true;
```

Comment:
Even if this line isn't reached, there is a possibility that some packages may have been pushed.
For instance, the chocolatey package was pushed, but NuGet package failed.
Or 1 of 2 NuGet packages failed, and so on.

Not sure if that is important though, but wanted to air it before doing anything.

Reply:
Hmm, that is a good point...

I think we would only want to sent notifications, it _all_ release packages were sent correctly.